apps/etterlatte-behandling/src/main/kotlin/oppgaveny/OppgaveRoutesNy.kt

```diff
@@ -67,5 +71,18 @@ internal fun Route.oppgaveRoutesNy(service: OppgaveServiceNy, kanBrukeNyOppgavel
                 call.respond(HttpStatusCode.Forbidden)
             }
         }
+        post("/opprett") {
+            if (kanBrukeNyOppgaveliste) {
+                val oppgaveRequest = call.receive<OpprettNyOppgaveRequest>()
+                service.opprettNyOppgaveMedSakOgReferanse(
+                    oppgaveRequest.referanse,
+                    oppgaveRequest.sakId,
+                    oppgaveRequest.oppgaveType
+                )
+                call.respond(HttpStatusCode.Created)
+            } else {
+                call.respond(HttpStatusCode.Forbidden)
+            }
+        }
```

Comment:
Jeg er ikke helt inne i loopen i oppgavehåndtering enda, så jeg er ikke helt klar på hele flyten -- men burde ikke det å opprette en attesteringsoppgave også lukke den eksisterende behandlingsoppgaven? 

Det virker som kan være greit med litt "sanity"-checking her hvis man tar med en referanse til den gamle oppgaven / servicen slår opp oppgaver for behandlingen og oppretter kun en attesteringsoppgave hvis den finner en åpen behandlingsoppgave (som også lukkes).

Reply:
Hadde egentlig tenkt til å ta det i senere oppgaver. Det er endel mer som må gjøres for å også kunne sende oppgaver tilbake til saksbehandler som hadde den originale saksbehandlingen etc